src/components/ItemEditorPage/RightPanel/Tags/Tags.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Props, State } from './Tags.types'
+import './Tags.css'
+
+const WHITE_SPACE_KEY_CODE = 32
+const ENTER_KEY_CODE = 13
+const COMMA_KEY_CODE = 118
+const BACKSPACE_KEY_CODE = 8
+const DELETE_KEY_CODE = 46
```

Comment:
It seems like these constants should live elsewhere, no? It seems out of place on the Tags Component

What do you think?